lbrynet/daemon/Daemon.py

```diff
@@ -1560,8 +2045,8 @@ def jsonrpc_channel_new(self, channel_name, amount):
 
         if amount <= 0:
             raise Exception("Invalid amount")
-        tx = yield self.wallet.claim_new_channel(channel_name, amount)
-        self.wallet.save()
+        tx = yield self.wallet_manager.claim_new_channel(channel_name, amount)
+        self.default_wallet.save()
```

Comment:
shouldn't the new wallet handle this? this was originally here because of a lbryum bug